### PR TITLE
AC Automated Test - LRAC 8821 | master

### PR DESCRIPTION
**Pull request by AfonsoHSantos**
https://issues.liferay.com/browse/LRAC-8821

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 1]
```diff
@@ -169,12 +169,21 @@ definition {
...
+		ACNavigation.goToSettings();
+
+		ACSettings.goToDataSources();
 
-		// TODO LRAC-8821 DataSourceListIsSearchable
+		ACUtils.searchBar(searchItem = "Liferay DXP");
 
+		AssertElementPresent(
+			itemName = "Liferay DXP",
+			locator1 = "ACSettings#ITEM_ON_LIST");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:186]: Hi @AfonsoHSantos, I think you can search for a data source that not present in the list

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase [hunk 2]
```diff
@@ -169,12 +169,21 @@ definition {
...
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToDataSources();
 
-		// TODO LRAC-8821 DataSourceListIsSearchable
+		ACUtils.searchBar(searchItem = "Liferay DXP");
 
+		AssertElementPresent(
+			itemName = "Liferay DXP",
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudConnectionTest.testcase:185]: Hi @AfonsoHSantos, this case will not work when running on CI. The Data Source name there is different from what we have running the test locally.

Please, take a look at this test `DataSourceList#DataSourceListShowsAllDataSources`, and see how to get the data source name.

**Comment by alexcandidos** [comment]
hi @AfonsoHSantos , please take a look at the suggestions and send a new PR when done!

Thanks!

**Comment by AfonsoHSantos** [comment]
Hi @alexcandidos @ViniLopes20, resent on #841

**Pull request closed without merging**